### PR TITLE
[filter/SNAP] Add TensorFlow Lite support

### PR DESCRIPTION
**Pull request by anyj0527**
- SNAP v3.0 supports inference with TensorFlow Lite.
- Add the supported enum.

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: ext/nnstreamer/tensor_filter/tensor_filter_snap.cc [hunk 1]
```diff
@@ -550,7 +550,9 @@ tensor_filter_snap::parse_custom_prop (const char *custom_prop, snap_option_s &s
       g_strstrip (option[1]);
 
       if (g_ascii_strcasecmp (option[0], "ModelFWType") == 0) {
-        if (g_ascii_strcasecmp (option[1], "TENSORFLOW") == 0) {
+        if (g_ascii_strcasecmp (option[1], "TENSORFLOWLITE") == 0) {
```
Comment (jaeyun-jung) [inline ext/nnstreamer/tensor_filter/tensor_filter_snap.cc:553]: let's fix property description also.
tensor_filter_snap::register_snap ()

Comment (anyj0527) [reply ext/nnstreamer/tensor_filter/tensor_filter_snap.cc:553]: Thanks. I updated the descrition.

**Review by jaeyun-jung — APPROVED** [review]

**Review by dongju-chae — APPROVED** [review]
LGTM!

**Comment by myungjoo** [comment]
@anyj0527 Question: is this backward compatible with SNAP v2.0?

**Review by again4you — APPROVED** [review]
LGTM!

**Comment by anyj0527** [comment]
> @anyj0527 Question: is this backward compatible with SNAP v2.0?

Yes. Only the enum for tensorflow lite is added in v3.0

**Comment by myungjoo** [comment]
> > @anyj0527 Question: is this backward compatible with SNAP v2.0?
> 
> Only the enum for tensorflow lite is added in v3.0. That causes compile error.

If this is the only update from v2.0, you really need to keep the backward compatibility. You can handle that.

**Comment by anyj0527** [comment]
> > > @anyj0527 Question: is this backward compatible with SNAP v2.0?
> > 
> > 
> > Only the enum for tensorflow lite is added in v3.0. That causes compile error.
> 
> If this is the only update from v2.0, you really need to keep the backward compatibility. You can handle that.

I updated the code to be built with those versions, PTAL.

**Review by myungjoo — APPROVED** [review]
LGTM

**Pull request merged**